src/styles/index.ts

```diff
@@ -2848,6 +2848,11 @@ const styles = (theme: ThemeColors) =>
             width: variables.iconSizeExtraLarge,
         },
 
+        billingBannerSubtitle: {
```

Comment:
Do we really need to write an ultra specific style here? Why not just reuse existing styles here? For example, we have `colorMuted` that can be added, and I bet we have our lineHeight as a style as well.

Reply:
Yes, we have `colorMuted`, but we do not  have styles that specify only the `lineHeight`

Reply:
Then let's add a generic helper class for lineHeight20... though I think our normal text should automatically have a good line height, right?

Reply:
Updated 
 
 Do you mean a specific style by a normal text?

Reply:
I'm saying that our standard, "default" text across the app should be at 15px, in our Expensify Neue font, with 20px line height. I'm not sure if all default text gets those styles or not though.

Reply:
Currently, we have 21px line height as a default
https://github.com/Expensify/App/blob/5c86b722a8df1c2cf11ee9d3a3e62c6e69a05376/src/styles/index.ts#L231C9-L231C22

Reply:
Got it. Let's just leave this as-is then, and I'll create a follow up issue to fix our line height from being 21px.

Reply:
reverted and removed specific line-height style